### YamlMime:UniversalReference
ms.openlocfilehash: 6cb0f3f80bc511a0865f3a5609cf1784ff077b9b
ms.sourcegitcommit: 87f95d58ec8de16e115bc344efeb084afc346b74
ms.translationtype: MT
ms.contentlocale: ja-JP
ms.lasthandoff: 08/12/2018
ms.locfileid: "40065807"
items:
- uid: azure-event-hubs.ErrorNameConditionMapper
  name: ErrorNameConditionMapper
  fullName: ErrorNameConditionMapper
  children:
  - azure-event-hubs.ErrorNameConditionMapper.ArgumentError
  - azure-event-hubs.ErrorNameConditionMapper.UnauthorizedError
  - azure-event-hubs.ErrorNameConditionMapper.ConnectionForcedError
  - azure-event-hubs.ErrorNameConditionMapper.ConnectionRedirectError
  - azure-event-hubs.ErrorNameConditionMapper.DecodeError
  - azure-event-hubs.ErrorNameConditionMapper.DetachForcedError
  - azure-event-hubs.ErrorNameConditionMapper.ErrantLinkError
  - azure-event-hubs.ErrorNameConditionMapper.FrameSizeTooSmallError
  - azure-event-hubs.ErrorNameConditionMapper.FramingError
  - azure-event-hubs.ErrorNameConditionMapper.HanldeInUseError
  - azure-event-hubs.ErrorNameConditionMapper.IllegalStateError
  - azure-event-hubs.ErrorNameConditionMapper.InternalServerError
  - azure-event-hubs.ErrorNameConditionMapper.InvalidFieldError
  - azure-event-hubs.ErrorNameConditionMapper.InvalidOperationError
  - azure-event-hubs.ErrorNameConditionMapper.LinkRedirectError
  - azure-event-hubs.ErrorNameConditionMapper.MessageLockLostError
  - azure-event-hubs.ErrorNameConditionMapper.MessageTooLargeError
  - azure-event-hubs.ErrorNameConditionMapper.ArgumentOutOfRangeError
  - azure-event-hubs.ErrorNameConditionMapper.NotImplementedError
  - azure-event-hubs.ErrorNameConditionMapper.OperationCancelledError
  - azure-event-hubs.ErrorNameConditionMapper.PreconditionFailedError
  - azure-event-hubs.ErrorNameConditionMapper.QuotaExceededError
  - azure-event-hubs.ErrorNameConditionMapper.ReceiverDisconnectedError
  - azure-event-hubs.ErrorNameConditionMapper.ResourceDeletedError
  - azure-event-hubs.ErrorNameConditionMapper.ResourceLockedError
  - azure-event-hubs.ErrorNameConditionMapper.SenderBusyError
  - azure-event-hubs.ErrorNameConditionMapper.ServerBusyError
  - azure-event-hubs.ErrorNameConditionMapper.ServiceCommunicationError
  - azure-event-hubs.ErrorNameConditionMapper.ServiceUnavailableError
  - azure-event-hubs.ErrorNameConditionMapper.SessionCannotBeLockedError
  - azure-event-hubs.ErrorNameConditionMapper.SessionLockLostError
  - azure-event-hubs.ErrorNameConditionMapper.SessionWindowViolationError
  - azure-event-hubs.ErrorNameConditionMapper.TransferLimitExceededError
  - azure-event-hubs.ErrorNameConditionMapper.UnattachedHandleError
  - azure-event-hubs.ErrorNameConditionMapper.MessagingEntityDisabledError
  langs:
  - typeScript
  type: enum
  summary: <span data-ttu-id="5c077-101">エラーの名前に、条件をマップします。</span><span class="sxs-lookup"><span data-stu-id="5c077-101">Maps the conditions to the Error names.</span></span>
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.ArgumentError
  name: ArgumentError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="5c077-102">正しくない引数が受信したときにエラーがスローされます。</span><span class="sxs-lookup"><span data-stu-id="5c077-102">Error is thrown when an incorrect argument was received.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.UnauthorizedError
  name: UnauthorizedError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="5c077-103">接続パラメーターが正しくないと、サーバーに接続が拒否されたときにエラーがスローされます。</span><span class="sxs-lookup"><span data-stu-id="5c077-103">Error is thrown when the connection parameters are wrong and the server refused the connection.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.ConnectionForcedError
  name: ConnectionForcedError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="5c077-104">演算子の intervened 何らかの理由により、接続を閉じるときにエラーがスローされます。</span><span class="sxs-lookup"><span data-stu-id="5c077-104">Error is thrown when an operator intervened to close the connection for some reason.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.ConnectionRedirectError
  name: ConnectionRedirectError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="5c077-105">コンテナーが現在の接続で使用可能でなくなったときにエラーがスローされます。</span><span class="sxs-lookup"><span data-stu-id="5c077-105">Error is thrown when the container is no longer available on the current connection.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.DecodeError
  name: DecodeError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="5c077-106">データをデコードできないときにエラーがスローされます。</span><span class="sxs-lookup"><span data-stu-id="5c077-106">Error is thrown when data could not be decoded.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.DetachForcedError
  name: DetachForcedError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="5c077-107">演算子は、何らかの理由をデタッチする intervened エラーがスローされます。</span><span class="sxs-lookup"><span data-stu-id="5c077-107">Error is thrown when an operator intervened to detach for some reason.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.ErrantLinkError
  name: ErrantLinkError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="5c077-108">エラーでデタッチされたリンクの入力を受け取ったときにエラーがスローされます。</span><span class="sxs-lookup"><span data-stu-id="5c077-108">Error is thrown when input was received for a link that was detached with an error.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.FrameSizeTooSmallError
  name: FrameSizeTooSmallError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="5c077-109">現在有効な値には、パフォーマティブの最小のエンコードに大きすぎて、合意済みの最大フレーム サイズのフレーム内に収まるようになるために、ピアは、フレームを送信できない場合は、エラーがスローされます。</span><span class="sxs-lookup"><span data-stu-id="5c077-109">Error is thrown when the peer cannot send a frame because the smallest encoding of the performative with the currently valid values would be too large to fit within a frame of the agreed maximum frame size.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.FramingError
  name: FramingError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="5c077-110">着信バイト ストリームから有効なフレーム ヘッダーを形成できないときは、エラーがスローされます。</span><span class="sxs-lookup"><span data-stu-id="5c077-110">Error is thrown when a valid frame header cannot be formed from the incoming byte stream.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.HanldeInUseError
  name: HanldeInUseError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="5c077-111">既に接続されているリンクの使用されているハンドルを使用して、アタッチが受信したときにエラーがスローされます。</span><span class="sxs-lookup"><span data-stu-id="5c077-111">Error is thrown when an attach was received using a handle that is already in use for an attached link.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.IllegalStateError
  name: IllegalStateError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="5c077-112">ピアが現在の状態で許可されていないフレームを送信エラーがスローされます。</span><span class="sxs-lookup"><span data-stu-id="5c077-112">Error is thrown when the peer sent a frame that is not permitted in the current state.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.InternalServerError
  name: InternalServerError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="5c077-113">内部サーバー エラーが発生したときにエラーがスローされます。</span><span class="sxs-lookup"><span data-stu-id="5c077-113">Error is thrown when an internal server error occured.</span></span> <span data-ttu-id="5c077-114">バグを発見したことがありますか。</span><span class="sxs-lookup"><span data-stu-id="5c077-114">You may have found a bug?</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.InvalidFieldError
  name: InvalidFieldError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="5c077-115">フレームの本文に渡された無効なフィールドと、操作を続行できませんでした、エラーがスローされます。</span><span class="sxs-lookup"><span data-stu-id="5c077-115">Error is thrown when an invalid field was passed in a frame body, and the operation could not proceed.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.InvalidOperationError
  name: InvalidOperationError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="5c077-116">操作が試行が許可されていません、エラーがスローされます。</span><span class="sxs-lookup"><span data-stu-id="5c077-116">Error is thrown when an operation is attempted but is not allowed.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.LinkRedirectError
  name: LinkRedirectError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="5c077-117">指定したアドレスは、現在のコンテナーでの終端に解決できないエラーがスローされます。</span><span class="sxs-lookup"><span data-stu-id="5c077-117">Error is thrown when the address provided cannot be resolved to a terminus at the current container.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.MessageLockLostError
  name: MessageLockLostError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="5c077-118">メッセージに対するロックが失われると、エラーがスローされます。</span><span class="sxs-lookup"><span data-stu-id="5c077-118">Error is thrown when the lock on the message is lost.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.MessageTooLargeError
  name: MessageTooLargeError
  children: []
  langs:
  - typeScript
  summary: '<span data-ttu-id="5c077-119">送信メッセージが大きすぎるときにエラーがスローされます。 最大サイズは 256 Kb です。</span><span class="sxs-lookup"><span data-stu-id="5c077-119">Error is thrown when the message sent is too large: the maximum size is 256Kb.</span></span>'
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.ArgumentOutOfRangeError
  name: ArgumentOutOfRangeError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="5c077-120">引数に値が許容される範囲外ですが、エラーがスローされます。</span><span class="sxs-lookup"><span data-stu-id="5c077-120">Error is thrown when an argument has a value that is out of the admissible range.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.NotImplementedError
  name: NotImplementedError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="5c077-121">機能はまだ実装されていませんが、プレース ホルダーが存在する場合に、エラーがスローされます。</span><span class="sxs-lookup"><span data-stu-id="5c077-121">Error is thrown when a feature is not implemented yet but the placeholder is present.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.OperationCancelledError
  name: OperationCancelledError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="5c077-122">サーバーは、内部の問題が原因で操作をキャンセルすると、エラーがスローされます。</span><span class="sxs-lookup"><span data-stu-id="5c077-122">Error is thrown when server cancels the operation due to an internal issue.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.PreconditionFailedError
  name: PreconditionFailedError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="5c077-123">操作を実行するために満たされる必要がある条件がなかった場合に、エラーがスローされます。</span><span class="sxs-lookup"><span data-stu-id="5c077-123">Error is thrown when a condition that should have been met in order to execute an operation was not.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.QuotaExceededError
  name: QuotaExceededError
  children: []
  langs:
  - typeScript
  summary: >-
    <span data-ttu-id="5c077-124">エラーがスローされます、Azure EventHub/ServiceBus クォータを超えました。</span><span class="sxs-lookup"><span data-stu-id="5c077-124">Error is thrown the the Azure EventHub/ServiceBus quota has been exceeded.</span></span>

    <span data-ttu-id="5c077-125">クォータのリセットは定期的に、この操作がそれまで待機する必要があります。</span><span class="sxs-lookup"><span data-stu-id="5c077-125">Quotas are reset periodically, this operation will have to wait until then.</span></span>

    <span data-ttu-id="5c077-126">メッセージング エンティティが最大許容サイズに達しています。</span><span class="sxs-lookup"><span data-stu-id="5c077-126">The messaging entity has reached its maximum allowable size.</span></span>

    <span data-ttu-id="5c077-127">これは、受信者の最大数 (5) がコンシューマー グループ レベルで既に開かれている場合に発生する可能性があります。</span><span class="sxs-lookup"><span data-stu-id="5c077-127">This can happen if the maximum number of receivers (which is 5) has already been opened on a per-consumer group level.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.ReceiverDisconnectedError
  name: ReceiverDisconnectedError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="5c077-128">2 つ以上のインスタンスが別のエポックの値を持つ同じパーティションに接続エラーがスローされます。</span><span class="sxs-lookup"><span data-stu-id="5c077-128">Error is thrown when two or more instances connect to the same partition with different epoch values.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.ResourceDeletedError
  name: ResourceDeletedError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="5c077-129">クライアントは連携サーバー エンティティが削除されたときにエラーがスローされます。</span><span class="sxs-lookup"><span data-stu-id="5c077-129">Error is thrown when a server entity the client is working with has been deleted.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.ResourceLockedError
  name: ResourceLockedError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="5c077-130">クライアントを与えませんアクセス別のクライアントが操作するためのサーバー エンティティを操作しようとしました。 エラーがスローされます。</span><span class="sxs-lookup"><span data-stu-id="5c077-130">Error is thrown when the client attempted to work with a server entity to which it has no access because another client is working with it.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.SenderBusyError
  name: SenderBusyError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="5c077-131">クライアントの送信者がメッセージを送信するための十分なリンク クレジットを持っていない場合に、エラーがスローされます。</span><span class="sxs-lookup"><span data-stu-id="5c077-131">Error is thrown when the client sender does not have enough link credits to send the message.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.ServerBusyError
  name: ServerBusyError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="5c077-132">サーバーがビジー状態の場合は、エラーがスローされます。</span><span class="sxs-lookup"><span data-stu-id="5c077-132">Error is thrown when the server is busy.</span></span> <span data-ttu-id="5c077-133">呼び出し元は、しばらくして、操作を再試行する必要があります。</span><span class="sxs-lookup"><span data-stu-id="5c077-133">Callers should wait a while and retry the operation.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.ServiceCommunicationError
  name: ServiceCommunicationError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="5c077-134">メッセージング操作に関連する一般的な通信エラーを通知するためのエラーです。</span><span class="sxs-lookup"><span data-stu-id="5c077-134">Error for signaling general communication errors related to messaging operations.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.ServiceUnavailableError
  name: ServiceUnavailableError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="5c077-135">サービスが利用できない場合は、エラーがスローされます。</span><span class="sxs-lookup"><span data-stu-id="5c077-135">Error is thrown when the service is unavailable.</span></span> <span data-ttu-id="5c077-136">操作を再試行する必要があります。</span><span class="sxs-lookup"><span data-stu-id="5c077-136">The operation should be retried.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.SessionCannotBeLockedError
  name: SessionCannotBeLockedError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="5c077-137">Azure service Bus のセッションをロックできないエラーがスローされます。</span><span class="sxs-lookup"><span data-stu-id="5c077-137">Error is thrown when the Azure ServiceBus session cannot be locked.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.SessionLockLostError
  name: SessionLockLostError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="5c077-138">Azure service Bus のセッションのロックが失われると、エラーがスローされます。</span><span class="sxs-lookup"><span data-stu-id="5c077-138">Error is thrown when the lock on the Azure ServiceBus session is lost.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.SessionWindowViolationError
  name: SessionWindowViolationError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="5c077-139">ピアのセッションの受信ウィンドウに違反した場合、エラーがスローされます。</span><span class="sxs-lookup"><span data-stu-id="5c077-139">Error is thrown when the peer violated incoming window for the session.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.TransferLimitExceededError
  name: TransferLimitExceededError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="5c077-140">ピアから送信されたメッセージの転送を複数のリンクで現在許可されているよりもエラーがスローされます。</span><span class="sxs-lookup"><span data-stu-id="5c077-140">Error is thrown when the peer sent more message transfers than currently allowed on the link.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.UnattachedHandleError
  name: UnattachedHandleError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="5c077-141">現在接続されているリンクの使用ではないハンドルを参照して、フレーム (アタッチ以外) が受信したときにエラーがスローされます。</span><span class="sxs-lookup"><span data-stu-id="5c077-141">Error is thrown when a frame (other than attach) was received referencing a handle which is not currently in use of an attached link.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.MessagingEntityDisabledError
  name: MessagingEntityDisabledError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="5c077-142">アクセス/無効になっているメッセージング エンティティに接続しようとしてエラーがスローされます。</span><span class="sxs-lookup"><span data-stu-id="5c077-142">Error is thrown when trying to access/connect to a disabled messaging entity.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
